components/avatar/avatar.html

```diff
@@ -0,0 +1,112 @@
+            Garuda UI
+            <ul class="nav-items">
+                <!-- <li><a class="anchor-style pointer" href="">Home</a></li> -->
+                <li><a class="nav-icons anchor-style pointer" href="/documentation/documentation.html">documentation</a>
+                </li>
+            </ul>
+            <div class="social">
+                <a href="https://github.com/tushargoyal812" class="nav-icons anchor-style"><i
+                        class="fab fa-github"></i></a>
+                <a href="https://twitter.com/tushargoyal29" class="nav-icons anchor-style"><i
+                        class="fab fa-twitter-square"></i></a>
+                <a href="https://www.linkedin.com/in/tushargoyal29/" class="nav-icons anchor-style"><i
+                        class="fab fa-linkedin"></i></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main-with-sidebar">
+        <div class="sidebar-wrapper">
+            <aside class="sidebar">
+                <h1>Get started</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Introduction</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Color</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Typography</a></li>
+                </ul>
+                <h1>Components</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links page-active" href="">Avatar</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Alert</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="/components/badge/badge.html">Badge</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Cards</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Image</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Input</a></li>
+                </ul>
+            </aside>
+        </div>
+        <main class="main-section">
+            <h1 class="headings">Avatar</h1>
+            <p class="heading-details">
+                Avatars are user profile pictures, usually found in circular shapes. Avatars can be used for 'container'
+                objects — projects, spaces, repositories, etcetera — to give them a recognisable visual identity.
+            </p>
+            <hr class="hr-margin">
+            <h2>Profile Avatar</h2>
+            <p class="heading-details">
+                Profile Avatars can be created using the class 'avatar', inside the img tag.
+            </p>
+            <section class="avatar-img">
+                <img class="avatar" src="/images/lady-avatar.jpg.crdownload" alt="avatar-1">
```

Comment:
Invalid filename. Please make sure images end with proper extension.

Replace partially downloaded files too.

Reply:
fixed it